[2025-04-13T20:07:55.837+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-13T20:07:55.858+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gcs_dlt_data_pipeline.download_file scheduled__1996-01-01T00:00:00+00:00 [queued]>
[2025-04-13T20:07:55.869+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gcs_dlt_data_pipeline.download_file scheduled__1996-01-01T00:00:00+00:00 [queued]>
[2025-04-13T20:07:55.870+0000] {taskinstance.py:2867} INFO - Starting attempt 3 of 2
[2025-04-13T20:07:56.343+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): download_file> on 1996-01-01 00:00:00+00:00
[2025-04-13T20:07:56.368+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'gcs_dlt_data_pipeline', 'download_file', 'scheduled__1996-01-01T00:00:00+00:00', '--job-id', '219', '--raw', '--subdir', 'DAGS_FOLDER/first_stage/first_stage.py', '--cfg-path', '/tmp/tmpawx75ipi']
[2025-04-13T20:07:56.383+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=229) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-13T20:07:56.399+0000] {standard_task_runner.py:72} INFO - Started process 246 to run task
[2025-04-13T20:07:56.397+0000] {standard_task_runner.py:105} INFO - Job 219: Subtask download_file
[2025-04-13T20:07:56.598+0000] {task_command.py:467} INFO - Running <TaskInstance: gcs_dlt_data_pipeline.download_file scheduled__1996-01-01T00:00:00+00:00 [running]> on host 7f7577d1d274
[2025-04-13T20:07:57.357+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='gcs_dlt_data_pipeline' AIRFLOW_CTX_TASK_ID='download_file' AIRFLOW_CTX_EXECUTION_DATE='1996-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__1996-01-01T00:00:00+00:00'
[2025-04-13T20:07:57.361+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-13T20:07:57.440+0000] {logging_mixin.py:190} INFO - Source File Path: raw/1996.csv.bz2
[2025-04-13T20:07:57.441+0000] {logging_mixin.py:190} INFO - Destination File Path: /tmp/1996.csv.bz2
[2025-04-13T20:07:57.442+0000] {logging_mixin.py:190} INFO - Downloading raw/1996.csv.bz2 from bucket db_project_data_bucket to /tmp/1996.csv.bz2
[2025-04-13T20:08:34.729+0000] {logging_mixin.py:190} INFO - Downloaded db_project_data_bucket/raw/1996.csv.bz2 to /tmp/1996.csv.bz2
[2025-04-13T20:09:30.305+0000] {local_task_job_runner.py:266} INFO - Task exited with return code -9. For more information, see https://***.apache.org/docs/apache-***/stable/troubleshooting.html#LocalTaskJob-killed
[2025-04-13T20:09:30.830+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-13T20:09:30.833+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
